### PR TITLE
Add action methods conventions;

### PR DESCRIPTION
**Pull request by Antondomashnev**
While I was reviewing the [PR](https://github.com/conichiGMBH/ios-guest/pull/773) on the iOS guest app. I noticed that @Mourad-Aly used another `action` methods naming. I have a proposal to finalize the notation we use in the app. 

@conichiGMBH/conichi-ios-devs looking forward to your thoughts 🚀

### CODE REVIEW

File: docs/swift_style_guide.md [hunk 1]
```diff
@@ -532,3 +532,18 @@ if name == "Hello" {
 Swift allows us to extend classes from `NSObject` to get Objective-C runtime features for an object. It also allows us to annotate Swift methods with `@objc` to allow the methods to be used by the Objective-C runtime.
 
 Supporting the Objective-C runtime means that method calls are going to be using dynamic dispatch instead of static or vtable dispatch. This end result is that methods that support the Objective-C runtime have a four times performance penalty when called. In practical usage, the performance hit may be negligible but the cool thing is that armed with this knowledge, we now know that method execution in Swift is four times faster than Objective-C.
+
+#### Actions
+
+The method which represents the action handling should be named with a verb in mind. There is not actually silver bullet for that, but since the 99% of the codebase in conichi use that style we should stick with that:
+
+```swift
+@IBAction func backButtonClicked(button: UIButton)
```
Comment (Antondomashnev) [inline docs/swift_style_guide.md]: We can pick between: `pressed`, `tapped`, `clicked`? 
@Mourad-Aly @wladyslaws let's vote =)

For me every work works fine, but if all you for `tapped` we can use `tapped`, but that we have to rename it through the app.

Comment (wladyslaws) [inline docs/swift_style_guide.md]: let's just use tapped from now on. It more resembles user's behavior.

Comment (Antondomashnev) [inline docs/swift_style_guide.md]: Deal, I'll update the PR

Comment (Antondomashnev) [inline docs/swift_style_guide.md]: Updated in 71d29af and 6571781 🚀

**Pull request merged**